coral-component-table/src/scripts/Table.js

```diff
@@ -1570,6 +1576,21 @@ class Table extends BaseComponent(HTMLTableElement) {
       headerCell.setAttribute('sortabledirection', column.sortableDirection);
       headerCell.setAttribute('aria-sort',
         column.sortableDirection === sortableDirection.DEFAULT ? 'none' : column.sortableDirection);
+
+      let liveRegion = this._elements.liveRegion;
+      if(liveRegion) {
```

Comment:
please add a space after if to make it consistent with the rest of your code

Reply:
done